public/main.js

```diff
@@ -112,14 +152,17 @@ function reloadTodoList() {
 
 function generateTodoListElement(todo) {
     var listItem = document.createElement("li");
+    listItem.className = "todo-item";
+    listItem.id = "todo-item" + todo.id;
     var todoText = document.createElement("p");
-    todoText.class = "todo-text";
+    todoText.className = "todo-text";
     todoText.id = "todo-text" + todo.id;
     todoText.textContent = todo.title;
     listItem.appendChild(todoText);
 
     var delBtn = document.createElement("button");
     delBtn.className = "del-btn";
+    delBtn.classList.add("button");
```

Comment:
Looks like the creation of the delete and update button are quite similar. See whether you can refactor them to use a common function.